src/FlashMessages.php

```diff
@@ -155,15 +159,17 @@ public function prolongFlash(): void
         }
     }
 
-    public function prepareMessages(SessionInterface $session, string $sessionKey): void
+    public function prepareMessages(SessionInterface $session): void
```

Comment:
Caution: this class is not final and this is a change of the public API.

Reply:
Oh... drat.